lint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+# With golangci-lint, if there are any compliation issues, then formatters' autofix won't be applied.
+# https://github.com/golangci/golangci-lint/issues/5257
+# However, running goimports first alone will actually fix some of the compilation issues.
+# Fixing formatting is also reasonable thing to do.
+# For this reason, this script runs golangci-lint in two stages:
+golangci-lint run --fix --no-config --disable-all --enable gofumpt,goimports $@
+exec golangci-lint run --fix $@
```

Comment:
How does this work on CI? We don't actually commit anything back, right?

Reply:
Neither lint.sh not "make lint" are called on CI. It uses golangci's github action instead: https://github.com/databricks/cli/blob/main/.github/workflows/push.yml#L75

Reply:
So we're not even going to modify anything in the worktree, because we don't pass --fix there. If there are any mismatches, they won't be corrected, the check will fail.